frontend/src/components/ErrorBoundary/ErrorBoundary.tsx

```diff
@@ -57,9 +71,14 @@ class ErrorBoundary extends React.Component<Props, State> {
               to inform us how the error occurred! Please also include brief description on the
               steps that led up to the error and a copy of the error messages seen below.
```

Comment:
Add info here to explain the difference between the two buttons. Probably something along the lines of "if you are seeing this page often or are seeing circles behave in some way you don't expect, try to reset your data. Otherwise you can return to circles to try again.

Reply:
Good idea